pages/wi/2021/jobs.js

```diff
@@ -142,17 +151,17 @@ const jobs = () => {
   return (
     <>
       <NextSeo
-        title="WI 2020 Jobs - THAT Conference"
+        title="WI 2021 Jobs - THAT Conference"
         description="Job openings for all WI 2020 THAT partners"
       />
 
       <ContentSection>
         <Main>
           <SideDetail>
-            <h1>THAT WI 2020 - Partner Jobs</h1>
+            <h1>THAT WI 2021 - Partner Jobs</h1>
             <p className="medium-body-copy">
               Full list of all the open opportunities at the amazing
-              organizations partnering with us to make THAT Wisconsin 2020
+              organizations partnering with us to make THAT Wisconsin 2021
```

Comment:
Possible to be dynamic?

Reply:
@brettski they are in pages/wi/2021/jobs....so dynamic doesn't really make sense.